app/components/object-inspector/property.js

```diff
@@ -54,10 +88,13 @@ export default Component.extend({
   },
 
   cannotEdit() {
+    if (this.model.name === '...') return true;
+    if (!this.isCalculated) return true;
     return this.isFunction || this.isOverridden || this.readOnly;
   },
 
   toggleDeps: action(function () {
+    if (!this.hasDependentKeys) return;
```

Comment:
Rather than returning for the false case, could we wrap in the truthy case before calling toggleProperty?